docs/adr/0021-make-dibbles-stata-typed-and-closed.md

```
@@ -2,11 +2,13 @@
 status: accepted
 ---
 
+> **Superseded in part by [ADR 0025](./0025-keep-gen-from-changing-the-container.md).** A tibble no longer becomes a dibble at its first `gen()`; `dibble()`, `as_dibble()`, and the readers are the only ways to get one.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)